src/ButtonGroup/ButtonGroup.features.stories.tsx

```diff
@@ -15,3 +16,47 @@ export const IconButtons = () => (
     <IconButton icon={DashIcon} aria-label="Subtract" />
   </ButtonGroup>
 )
+
+export const IconButtonsWithTooltip = () => (
```

Comment:
Happy to remove them after the review if we don't want to keep them

Reply:
I think we can leave em' :)